src/domain/consents/generateChallenge.ts

```diff
@@ -58,7 +59,12 @@ export async function updateConsentCredential (
 
   // Update in database,
   // relying on database validation for any null or relational aspects.
-  await consentDB.update(consent)
+  try {
+    await consentDB.update(consent)
+  } catch (error) {
+    throw new DatabaseError(consent.id)
```

Comment:
you should log `error` here elsewhere we will know nothing about its nature

Reply:
agreed